applications/asset_tracker_v2/src/main.c

```diff
@@ -529,6 +529,13 @@ void on_sub_state_passive(struct app_msg_data *msg)
 			passive_mode_timers_start_all();
 		}
 	}
+	if (IS_EVENT(msg, sensor, SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED)
+	    && k_timer_remaining_get(&movement_resolution_timer) != 0) {
+		data_sample_timer_handler(NULL);
+		/* Trigger a sample request if there has been inactivity after
```

Comment:
nit: Typically you find comments above the action that is performed. It would be nice if this could be done here as well.